packages/trpc/server/routers/viewer/slots/reserveSlot.handler.ts

```diff
@@ -42,8 +43,12 @@ export const reserveSlotHandler = async ({ ctx, input }: ReserveSlotOptions) =>
   // If this is a seated event then don't reserve a slot
   if (eventType.seatsPerTimeSlot) {
     // Check to see if this is the last attendee
-    const bookingWithAttendees = await prisma.booking.findUnique({
-      where: { uid: bookingUid },
+    const bookingWithAttendees = await prisma.booking.findFirst({
+      where: {
+        eventTypeId,
+        startTime: slotUtcStartDate,
+        endTime: slotUtcEndDate,
+      },
```

Comment:
You can see here (https://github.com/calcom/cal.com/blob/main/packages/prisma/schema.prisma#L800), that Booking is indexed on startTime, endTime and status. So, adding status: 'ACCEPTED' would allow this query to hit index. Love this approach!